src/site/content/en/blog/deprecating-excalidraw-electron/index.md

```diff
@@ -0,0 +1,267 @@
+that runs in its own window. It is fully integrated in the operating system's multitasking UI and
+gets its own app icon on the home screen, Dock, or task bar; depending on what platform you install
+it.
+
+<figure class="w-figure">
+  <img src="excalidraw-pwa.png" alt="Excalidraw running in its own window.">
+  <figcaption class="w-figcaption">The Excalidraw PWA in a standalone window</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="excalidraw-icon.png" width="400" alt="Excalidraw icon on the macOS Dock.">
+  <figcaption class="w-figcaption">The Excalidraw icon on the macOS Dock</figcaption>
+</figure>
+
+### File system access
+
+Excalidraw makes use of [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs) for
+accessing the file system of the operating system. On supporting browsers, this allows for a true
+open→edit→save workflow and actual over-saving and "save as", with a transparent fallback for
+other browsers. You can learn more about this feature in my blog post
+[Reading and writing files and directories with the browser-nativefs library](browser-nativefs/).
+
+### Drag and drop support
+
+Files can be dragged and dropped onto the Excalidraw window just as in native applications. On a
+browser that supports the [File System Access API](https://web.dev/file-system-access/), a dropped
+file can be immediately edited and the modifications be saved to the original file. This is so
+intuitive that you sometimes forget that you are dealing with a Web app.
+
+### Clipboard access
+
+Excalidraw works well with the operating system's clipboard. Entire Excalidraw drawings or also just
+individual objects can be copied and pasted in `image/png` and `image/svg+xml` formats, allowing for
+an easy integration with other native tools like [Inkscape](https://inkscape.org/) or Web-based
+tools like [SVGOMG](https://jakearchibald.github.io/svgomg/).
+
+<figure class="w-figure">
+  <img src="clipboard.png" alt="Excalidraw context menu showing the 'copy to clipboard as SVG' and 'copy to clipboard as PNG' menu items.">
+  <figcaption class="w-figcaption">The Excalidraw context menu offering clipboard actions</figcaption>
+</figure>
+
+### File handling
+
+Excalidraw already supports the experimental [File Handling API](https://web.dev/file-handling/),
+which means `.excalidraw` files can be double-clicked in the operating system's file manager and
+open directly in the Excalidraw app, since Excalidraw registers as a file handler for `.excalidraw`
+files in the operating system.
+
+### Declarative link capturing
+
+Excalidraw drawings can be shared by link—here is an
+[example](https://excalidraw.com/#json=4646308765761536,jwZJW8JsOM75vdhqG2nBgA). In the future, if
+people have Excalidraw installed as a PWA, such links will not open in a browser tab, but launch a
+new standalone window. This will work thanks to
+[declarative link capturing](https://github.com/WICG/sw-launch/blob/master/declarative_link_capturing.md),
```

Comment:
Saying this will be implemented is premature. There isn't even an intent to prototype. This statement is also tantamount to saying we'll implement it whether the other browser vendors do or not.

Reply:
I added "Pending implementation" at the beginning. A lot of these features do not work in all browsers, but the thing is: they are a progressive enhancement. In the concrete case, rather than opening in a new PWA window, a link would just open in a browser tab, even if the PWA is installed. No harm done.